__tests__/api.test.js

```diff
@@ -0,0 +1,22 @@
+const app = require("../app");
+const request = require("supertest");
+
+describe("GET /api", () => {
+  test("status:200, respondes with a list of description of all the currently released endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body);
```

Comment:
Make sure you remove console.logs

Reply:
Yes, I have removed in the lasted commit. Can I merge?